strawberry/federation/scalar.py

```diff
@@ -10,7 +10,7 @@
     _T = TypeVar("_T", bound=type)
 
 
-def identity(x):  # pragma: no cover
+def identity(x) -> Any:  # pragma: no cover
```

Comment:
same as above, maybe we can reuse the one from above instead of declaring it again?